src/sentry/models/notificationsetting.py

```diff
@@ -47,8 +74,21 @@ def value_str(self) -> str:
     def provider_str(self) -> str:
         return get_provider_name(self.provider)
 
+
+@region_silo_only_model
+class NotificationSetting(Model, NotificationSettingBase):
+    """
+    A setting of when to notify a user or team about activity within the app.
+    Each row is a notification setting where a key is:
+    ("scope_type", "scope_identifier", "target", "provider", "type"),
+    and the value is ("value").
+    """
+
+    __include_in_export__ = False
+
     scope_type = BoundedPositiveIntegerField(
         choices=(
+            # TODO:actor remove user scope type from region model
```

Comment:
👍 This is really, really cool work btw.  Sentry is going to benefit from this change in a general way.